stdlib/public/TensorFlow/Ops.swift

```diff
@@ -372,6 +372,23 @@ public extension Tensor where Scalar : Numeric {
   }
 }
 
+/// Performs batched matrix multiplication with another tensor and produces the
+/// result.
+@inlinable @inline(__always)
+@differentiable(
+  wrt: (lhs, rhs),
+  vjp: _vjpBatchedMatmul
+  where Scalar : Differentiable & FloatingPoint
+)
+public func batchedMatmul<Scalar : Numeric>(
```

Comment:
I think `batchMatmul` is better and more familiar. It's top-level math function, so we don't have to worry too much about twisting the verb form to make it look swifty.

Reply:
Amusingly, MKL calls this `batch_gemm` while cuBLAS calls it `gemmBatched`. I do think that `batchedMatmul` is slightly clearer (the operation is itself batched, not just the inputs) but I don't care all that much one way or the other.

Reply:
WFM.